src/cmds/categories/create.js

```diff
@@ -0,0 +1,127 @@
+const { debug } = require('../../lib/logger');
+
+module.exports = class CategoriesCreateCommand {
+  constructor() {
+    this.command = 'categories:create';
+    this.usage = 'categories:create <title> [options]';
+    this.description = 'Create a category with the specified title and guide in your ReadMe project';
+    this.category = 'categories';
+    this.position = 2;
+
+    this.hiddenargs = ['title'];
+    this.args = [
+      {
+        name: 'key',
+        type: String,
+        description: 'Project API key',
+      },
+      {
+        name: 'version',
+        type: String,
+        description: 'Project version',
+      },
+      {
+        name: 'title',
+        type: String,
+        defaultOption: true,
+      },
+      {
+        name: 'categoryType',
+        type: String,
+        description: 'Category type, must be guide or reference',
+      },
+      {
+        name: 'preventDuplicates',
+        type: Boolean,
+        description:
+          'Prevents the creation of a new category if their is an existing category with a mathcing `categoryType` and exact matching `title`',
+      },
+    ];
+  }
+
+  async run(opts) {
+    const { categoryType, title, key, version, preventDuplicates } = opts;
+    debug(`command: ${this.command}`);
+    debug(`opts: ${JSON.stringify(opts)}`);
+
+    if (!key) {
+      return Promise.reject(new Error('No project API key provided. Please use `--key`.'));
+    }
+
+    if (!title) {
+      return Promise.reject(new Error(`No title provided. Usage \`${config.get('cli')} ${this.usage}\`.`));
+    }
+
+    if (categoryType !== 'guide' && categoryType !== 'reference') {
+      return Promise.reject(new Error('`categoryType` must be guide or reference.'));
+    }
+
+    const selectedVersion = await getProjectVersion(version, key, false);
+
+    debug(`selectedVersion: ${selectedVersion}`);
+
+    function getNumberOfPages() {
+      return fetch(`${config.get('host')}/api/v1/categories?perPage=20&page=1`, {
+        method: 'get',
+        headers: cleanHeaders(key, {
+          'x-readme-version': selectedVersion,
+          Accept: 'application/json',
+        }),
+      }).then(res => {
+        return Math.ceil(res.headers.get('x-total-count') / 20);
+      });
+    }
+
+    const allCategories = [].concat(
+      ...(await Promise.all(
+        Array.from({ length: await getNumberOfPages() }, (_, i) => i + 1).map(async page => {
+          return fetch(`${config.get('host')}/api/v1/categories?perPage=20&page=${page}`, {
+            method: 'get',
+            headers: cleanHeaders(key, {
+              'x-readme-version': selectedVersion,
+              Accept: 'application/json',
+            }),
+          }).then(res => handleRes(res));
+        })
+      ))
+    );
+
+    async function matchCategory() {
+      return allCategories.filter(category => {
+        return changeCase.paramCase(category.title) === changeCase.paramCase(title) && category.type === categoryType;
```

Comment:
Do we still need `changeCase` here?
```suggestion
        return category.title === title && category.type === categoryType;
```

Reply:
@kanadgupta I used the changeCase library here to ensure matches on capitalization and spaces. As to ensure an exact match. Open to other methods here.

Reply:
I'm comfortable just doing a `.toLowerCase()` check on these instead of pulling `change-case` in for this one line.

Reply:
Also if you could add a test case (i.e. with "Example Category" and "Example category "), that'd be great!